lab/tools.py

```diff
@@ -13,19 +13,7 @@
 import sys
 from pathlib import Path
 
-# Use simplejson where it's available, because it is compatible (just separately
-# maintained), puts no blanks at line endings and loads json much faster:
-# json_dump: 44.41s, simplejson_dump: 45.90s
-# json_load: 7.32s, simplejson_load: 2.92s
-# We cannot use cjson or ujson for dumping, because the resulting files are
-# hard to read for humans (cjson_dump: 5.78, ujson_dump: 2.44). Using ujson for
-# loading might be feasible, but it would only result in a very small speed gain
-# (ujson_load: 2.49). cjson loads even slower than simplejson (cjson_load: 3.28).
-try:
-    import simplejson as json
-except ImportError:
-    import json
-
+import orjson as json
```

Comment:
```suggestion
import orjson
```

since it's not compatible with the json module.